app/controllers/tasks_controller.rb

```diff
@@ -6,12 +6,19 @@ class TasksController < ApplicationController
   # GET /tasks
   # GET /tasks.json
   def index
-    @tasks = Task.all
+    @completion = Task.completion 
```

Comment:
Is there a more descriptive name we could use for this variable? Or what it is used for?